src/configuration/configuration.ts

```diff
@@ -0,0 +1,37 @@
+import { ethers } from "ethers";
+import * as zAuction from "@zero-tech/zauction-sdk";
+
+export const zAuctionConfiguration = (
+  web3Provider: ethers.providers.Web3Provider,
+  apiUri?: string,
+  subgraphUri?: string,
+  zAuctionAddress?: string,
+  tokenContract?: string
+): zAuction.Config => {
+  let defaultApiUri;
+  let defaultSubgraphUri;
+  let defaultZAuctionAddress;
+  let defaultTokenContract;
+
+  if (web3Provider.network.name == "mainnet") {
+    defaultApiUri = "https://mainnet.zauction.api.zero.tech/api";
+    defaultSubgraphUri =
+      "https://api.thegraph.com/subgraphs/name/zer0-os/zauction";
+    defaultZAuctionAddress = "0x05cBD37cA528B7ea50800aA80ddD0F9F30C952F0";
+    defaultTokenContract = "0xc2e9678A71e50E5AEd036e00e9c5caeb1aC5987D";
+  } else {
```

Comment:
I think it'd be worth it to add some validation around non-mainnet networks. If, in the future, I want to specifiy that we should use rinkeby, and pass it into the constructor, there won't be any sort of warning that I'm actually on kovan.

I think we should just have some sort of ```else { throw new Error(`Network ${web3Provider.network.name} not supported`); }``` to avoid that scenario.